code/items/equipments/entities/C4Entity.cs

```diff
@@ -12,28 +13,62 @@ namespace TTTReborn.Items
     [Library("ttt_c4_ent"), Hammer.Skip]
     public partial class C4Entity : Prop, IUse
     {
+        public static C4Preset[] TimerPresets =
+        {
+            new C4Preset
+            {
+                timer = 60,
+                wires = 1
+            },
+
+            new C4Preset
+            {
+                timer = 150,
+                wires = 2
+            },
+
+            new C4Preset
+            {
+                timer = 300,
+                wires = 4
+            }
```

Comment:
json based would be cool :D But that's okay as well

Reply:
for customizability? I think this will be fine for a while.

Reply:
Disagree with json. I like this.

Reply:
As I've mentioned before, I'd much rather prefer all data be stored in JSON and all behavior in C#. 
This looks clunky and clutters the C# file which should be used for behavior, not pure data storage.
To solve one of the two issues, we could extract this into a separate file.